content/backmatter.tex

```diff
@@ -496,25 +519,61 @@ \subsection{\textit{Fortran} API}\label{subsec:deprecate-fortran} %% WARNING: Is
 \footnote{Formally, \Fortran[2003] is known as ISO/IEC~1539-1:2004(E).}.
 
 
-\subsection{Active-set-based collective routines}
-With the addition of \openshmem teams, the previous methods for performing collective
+\subsection{Active-set-based collectives}
+With the addition of \hyperref[subsec:team]{\openshmem teams}, the previous method for performing collective
```

Comment:
I think this styling doesn't quite fit our general use of references. Typically, we don't hyperlink text; we reference "Section 7.2." So, I'd favor that use here.

Reply:
I don't mind changing this sentence with Section \ref. The same hyperlink is in the deprecation table though. Should I do the same change there or leave it as is?

Reply:
Err, by "same" I mean there's a similar hyperlink to `\ref{subsec:team_collectives}` for the "Team-based collectives" in the replacement column. Text pointing a section.

Reply:
I'll retract this comment. The backmatter uses `\hyperref` a lot. All but two uses in the entire document are in the backmatter, so I'm fine leaving this as is.

Reply:
I came around to it and now agree with you. I made the change in https://github.com/BryantLam/openshmem-specification/pull/14.